sql/fxa_users_daily_v1.sql

```diff
@@ -18,14 +18,29 @@ CREATE TEMP FUNCTION udf_mode_last(x ANY TYPE) AS ((
   LIMIT 1
 ));
 
+CREATE TEMP FUNCTION udf_contains_tier1_country(x ANY TYPE) AS ((
```

Comment:
is the double paren indicating a subquery expression needed/valid here?

Reply:
I forget the exact terminology for it, but I believe the double paren actually serves a function here, defining this as an expression that returns a single value. We have to do the same for the udf_mode_last function.

Reply:
the terminology is scalar subquery expression, but normally it's followed by `SELECT`, but here it's not because it's using `EXISTS()`.

Reply:
also, I can confirm it works with one less set of parens:
```sql
CREATE TEMP FUNCTION udf_contains_tier1_country(x ANY TYPE) AS (
  EXISTS(
    SELECT
      country
    FROM
      UNNEST(x) AS country
    WHERE country IN (
      'United States',
      'France',
      'Germany',
      'United Kingdom',
      'Canada')
    ));
SELECT udf_contains_tier1_country(['??', 'United States', '??'])
```
returns `true`